receiver.go

```diff
@@ -471,6 +471,18 @@ func newReceiver(source string, s *Session, opts *ReceiverOptions) (*Receiver, e
 		l.l.receiverSettleMode = opts.SettlementMode
 	}
 	l.l.target.Address = opts.TargetAddress
+	for _, v := range opts.SenderCapabilities {
+		l.l.source.Capabilities = append(l.l.source.Capabilities, encoding.Symbol(v))
```

Comment:
`l.l` could become `r.l`. There should be a name for this pattern where you have a char.<samechar>, like there is for dogsled. ;)